model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -759,6 +757,8 @@ case class AzureManagedAppCoordinates(tenantId: UUID,
                                       landingZoneId: Option[UUID] = None
 )
 
+case class GcpProjectInfo(projectId: GoogleProjectId)
```

Comment:
`GoogleProjectId` is already a wrapper around `String` for type safety - do we actually need this additional layer of wrapper?

If we do need it, I note that we don't have a `JsonFormat` for `GcpProjectInfo`, so we would likely fail to de/serialize it in JSON requests/responses. I don't think `GcpProjectInfo` is used anywhere that needs serdes, so this is ok, just raising it to keep an eye on

Reply:
I'd love to avoid touching this file, although I thought maybe we'd eventually (YAGNI?) cram more stuff into `GcpProjectInfo`...is this potentially a landmine for the next person if I leave it in here without `JsonFormat`?

Reply:
Ok dropped it...very happy with the outcome!  Thanks for the suggestion.